fusion-cli/build/load-fusionrc.js

```diff
@@ -16,10 +16,16 @@ const chalk = require('chalk');
 let loggedNotice = false;
 
 /*::
+
+type CompileResult = {
```

Comment:
Do you think the following would be a valid use case?

A module that should be bundled in both the server and client.
But `transpile: "spec"` in the browser, and `transpile: "none"` in the server.

I'm thinking probably not. In theory, the should be no reason to do `transpile: "spec"` for both, assuming the parse is already happening once. Just wanted to make note of the reasoning for posterity.

Reply:
Seems like a valid use case to me, since the module can have completely different source code for the server and browser bundles via different entry points.

Reply:
Okay, unless I'm missing something, this is impossible to express with the current API. Since it seems the `transforms`/`transpile` option would apply to both server and client?